### PR TITLE
Heatmap ui tweaks

### PR DESCRIPTION
**Pull request by gitjul**
- [x] Use proper grey in colors.css
- [x] Style heat map inputs
- [x] Tweak slider to fix width issues
- [x] Style "reset to default" button in heat map
- [x] Add alts to images in elm

### CODE REVIEW

File: app/assets/stylesheets/config/typography.css.scss [hunk 1]
```diff
@@ -4,7 +4,6 @@ body {
   color: black;
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased;
-  font-smoothing: antialiased;
```
Comment (gitjul) [inline app/assets/stylesheets/config/typography.css.scss]: there's no such property (there's [`font-smooth`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth) but it's non standard and I wouldn't use it.)

btw there are reason's not to use smoothing, it's not always the best idea, e.g. contrast is worse
- http://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/
- https://www.zachleat.com/web/font-smooth/

File: app/assets/stylesheets/modules/buttons-links.css.scss [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
   font-weight: $font-stack-bold;
+  height: 33px;
+  text-align: center;
 }
 
-.circular-button {
+.link-button {
   height: 33px;
+  margin-top: 3px;
   width: 33px;
```
Comment (gitjul) [inline app/assets/stylesheets/modules/buttons-links.css.scss]: this is pretty specific and there's nothing else using this class now, so I'm merging those two classes

File: app/assets/stylesheets/modules/buttons-links.css.scss [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
-}
 
-.link-button {
-  marign-top: 3px;
+  img {
+    vertical-align: middle;
```
Comment (gitjul) [inline app/assets/stylesheets/modules/buttons-links.css.scss]: fixes alignment. note: changing line-height messes with this, but there's no reason to set line-height if no text

File: app/assets/stylesheets/config/typography.css.scss (hunk 1 above)
Comment (3v0k4) [reply app/assets/stylesheets/config/typography.css.scss]: Cool stuff! Thanks for sharing

**Pull request merged**